Assets/Scripts/Manager/StorySceneManager.cs

```diff
@@ -0,0 +1,577 @@
+		human.layer = LayerMask.NameToLayer ("Doll"); //make human layer to doll so can ghost switch into
+
+		//Puppet master Showdown!
+		Coroutine movecam = StartCoroutine (move (maincam.transform, puppetMaster.transform, 0.04f));
+		yield return new WaitForSeconds(0.7f);
+		StopCoroutine (movecam);
+		puppetMaster.SetActive (true);
+		puppetMasterFace("forward");
+		yield return new WaitForSeconds(1.5f);
+
+		// "Congratulations!..."
+		showText(0);
+		while (textBoxManager.isActive) {
+			yield return null;
+		}
+
+		// simulate ghost switch from player to human
+		momentaryJail.SetActive(true); // prevent human from moving around
+		puppetMasterFace("right");
+		puppetMasterAnim.SetBool("raiseHand", true);
+		movecam = StartCoroutine (move (maincam.transform, human.transform, 0.04f));
+		StartCoroutine(simulateGhostSwitch (player, human));
+		yield return new WaitForSeconds(0.8f);
+		puppetMasterAnim.SetBool("raiseHand", false);
+		//puppetMaster face forward
+		puppetMasterFace("forward");
+
+		humanController.enabled = false;
+		player.layer = 0;
+		playerAnim.enabled = true;
+		playerAnim.SetTrigger ("Sliced");
+		StopCoroutine (movecam);
+
+
+		yield return new WaitForSeconds(1f);
+		// "Tada! Just as promised.."
+		showText(1);
+		while (textBoxManager.isActive) {
+			yield return null;
+		}
+
+		StartCoroutine (thirdTrigger ());
+		yield return null;
+	}
+
+	IEnumerator thirdTrigger(){
+
+		musicManager.playTrack (2); //E3
+		yield return new WaitForSeconds(0.5f);
+		musicManager.stopTrack (1); //E2
+		// open the door with light and stuff
+		doorToOpen.SetActive (true);
+		humanController.enabled = true;
+		humanController.allowSound = false;
+		yield return new WaitForSeconds(5f); // allow human to escape //TODO
+		humanController.enabled = false;
+
+		//puppet Master face human and starts moving closer
+		puppetMasterFace("right");
+		string[] array = textBoxManager.GetComponent<DialogueParser>().textArrays[2].Split (';');
+		for (int i = 0; i < 3; i++) {
+			//puppetmaster inches closer
+			StartCoroutine (simulatePuppetMasterWalking(new Vector3 (5f - 1f*i, 0, 0), 1f));
+			textBoxManager.reloadText (array[i]);
+			textBoxManager.EnableTextBox();
+			while (textBoxManager.isActive) {
+				yield return null;
+			}
+		}
+
+		// “I mean, look at just how adorable I am!”
+		showText(3);
+		while (textBoxManager.isActive) {
+			yield return null;
+		}
+
+		StartCoroutine (simulatePuppetMasterWalking(new Vector3 (2, 0, 0), 0.9f));
+
+		// "But I’ve been waiting soooo long…"
+		showText(4);
+		while (textBoxManager.isActive) {
+			yield return null;
+		}
+
+		puppetMasterFace("forward");
+		StartCoroutine (askPuppetMasterName());
+		while (!isNameUpdated) {
+			yield return null;
+		}
+		musicManager.stopTrack(2); //E3
+		yield return new WaitForSeconds(0.4f);
+		// need to use GameManager.getInstance ().isTrueEnding () instead
+		if (puppetMasterName == "annie" || puppetMasterName == "Annie" || puppetMasterName == "ANNIE") {
```

Comment:
can you change this to puppetMasterName.toLower (to change to lowercase) == "annie"